src/dataExplorer/components/QueryTime.scss

```diff
@@ -2,6 +2,7 @@
 
 .query-status {
   flex-grow: 1;
+  margin-top: 14px;
```

Comment:
hoping to remove this soon to be more inline with clockface styling, but had to shoehorn it in